Changelog

```diff
@@ -1,3 +1,45 @@
+Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.0.0
+    - FEATURE: Drop support for Qt5, qmake, autotools, Windows < 10, Windows 32-bit
+    - FEATURE: Support creating .torrent with larger piece size (Chocobo1) 4d2015cfeda95be0fd9f6f9628ab77d48c741d17
+    - FEATURE: Improve tracker entries handling (glassez) 7cd2445a49e879255dca5015592ea94ccd9d0d12
+    - FEATURE: Add separate filter item for tracker errors (glassez) 3ca093941108c27ee24ad984877baf0ee17d961e
+    - FEATURE: Allow to remove tracker from tracker filter widget menu (glassez) 30189ae943b9a85d39484f6cdc64e2200518654f
+    - FEATURE: Implement `Reannounce In` column (Hanabishi) c394868f87e71d8ab614c6ae5463c712acffd172
+    - FEATURE: Expose `DHT bootstrap nodes` setting (Chocobo1) dcba9eda009fcb74e8aebbee4a4c5df7248abf06
+    - FEATURE: Improve GUI behavior when adding multiple torrents (glassez) 70b438e6d9249bacf8589ba50d16bbfd617fd199
+    - FEATURE: Add support for [Mark-of-the-Web](https://redcanary.com/threat-detection-report/techniques/mark-of-the-web-bypass/) (Chocobo1) 34f7b75f12390454ed831037ff34531b90307c55
+    - FEATURE: Keep unwanted files in separate folder (configurable) (glassez) e216a9cf53ae1203d148a013ab5d71d6672fe97a
+    - FEATURE: Add `Copy Comment` to the torrent list's context menu (thalieht) 30d9978c97a582b396c53441c938b05b124ba8c9
+    - FEATURE: Allow relative profile paths (Victor Chernyakin) 0297f0f34baa51cde6087edd2f521f108109ad2e
+    - FEATURE: Enable Ctrl+F hotkey for more inputs (thalieht) 8dcc734df562885131dbd0c2cc4ec31e24361559
+    - FEATURE: Add seeding limits to RSS and Watched folders options UI (glassez) d652a10495fdd5175242f029a7e2b9a6373ea985
+    - BUGFIX: Display error message when unrecoverable error occurred (glassez) a0e41a11dec9e195455d990fd0e87390882c703d
+    - BUGFIX: Update size of selected files when selection is changed (glassez) eddfaedc4d34cfcaf8a38210e6700ace0886cc9f
+    - BUGFIX: Specify a locale if none is set (Chocobo1) fc56fdade75610fabf42426cdc8960b9f1ba77a4
+    - WEBUI: Improve WebUI responsiveness (Chocobo1) 2edb1a0765dc501bb66d133d1bc8ac3c37704f51
+    - WEBUI: Add I2P settings to WebUI (thalieht) a3888811f6ad5a82119805378730a68402765bfa
+    - WEBUI: Do not exit the app when WebUI hass failed to start (Hanabishi) 35e4b35d125983051a25173d595119d99b7bda3a
+    - WEBUI: Add `Moving` filter to side panel (xavier2k6) f067ab169224013cb4bf21630b8a3eaa5d7d2dd6
+    - WEBUI: Add dark theme (d47081) 9d90141c29b80b03631e19cba0fa6f1cddc30423
+    - WEBUI: Allow to remember torrent content files deletion (David) e69f85782881a4b831ed97694263f027ecfa568d
+    - WEBUI: Leave the fields empty when value is invalid (Chocobo1) 54bcabeca4292a72649202a03eaf09e711d73395
+    - WEBUI: Use natural sorting (Chocobo1) 00372dd559dedd19e3bcb66db8e7d5faf597d4b2
+    - WEBUI: WebAPI: Get torrent count  (glassez) cecbb44a031c96e8fce48f12eeddda47b9c85a27
```

Comment:
`WEBUI: WebAPI:` looks ugly.

Reply:
Then maybe it's time I use `WEBAPI` as a category